Makefile

```diff
@@ -511,6 +512,7 @@ docker-compose-sources: .git/hooks/pre-commit
 	fi;
 
 	ansible-playbook -i tools/docker-compose/inventory tools/docker-compose/ansible/sources.yml \
+			-e sources_dest=$(abspath $(SOURCES)) \
```

Comment:
indentation kind of weird

Reply:
Had to turn on whitespace. The one at the very bottom had all spaces. Looks like the rest of the file is a tab for command blocks and then spaces. I think I am used to (and my editor is used to) just doing all tabs for the command blocks. Think I fixed it.